databroker/core.py

```diff
@@ -384,14 +443,15 @@ def get_table(mds, fs, headers, fields=None, stream_name='primary', fill=False,
         return pd.DataFrame()
 
 
-def restream(mds, fs, headers, fields=None, fill=False):
+def restream(mds, fs, es, headers, fields=None, fill=False):
```

Comment:
I agree, this would be replaced by `Header.events` (which I note is a bad name, maybe `Header.documents`?